src/engine/attributes/string.js

```diff
@@ -13,13 +16,13 @@ export default function stringApply(val, userVal, op) {
     } else if (op == "NEQ") {
         return uv != v
     } else if (op == "SW") {
-        return startsWith(uv, v)
+        return uv.startsWith(v);
     } else if (op == "NSW") {
-        return !startsWith(uv, v)
+        return !uv.startsWith(v);
     } else if (op == "EW") {
-        return endsWith(uv, v, true)
+        return uv.endsWith(v);
```

Comment:
The endsWith() method is not supported in IE 11 (and earlier versions).

Reply:
would that only be true for javascript? This is back-end nodejs server.

Reply:
Correct!